app/controllers/tpi/sectors_controller.rb

```diff
@@ -112,7 +113,7 @@ def companies_scope(params)
         TPISector.friendly.find(params[:id]).companies.published
       else
         Company.published
-      end
+      end.active
```

Comment:
This look a bit strange and can be easily overlooked during future changes. If we add `published_active` scope (see prev comment) we won't need that here :)